diff --git a/node_modules/webrtc-adapter/src/js/common_shim.js b/node_modules/webrtc-adapter/src/js/common_shim.js
index 96d7c8e..063ea64 100644
--- a/node_modules/webrtc-adapter/src/js/common_shim.js
+++ b/node_modules/webrtc-adapter/src/js/common_shim.js
@@ -388,7 +388,7 @@ export function shimAddIceCandidateNullOrEmpty(window, browserDetails) {
   }
   const nativeAddIceCandidate =
       window.RTCPeerConnection.prototype.addIceCandidate;
-  if (!nativeAddIceCandidate || nativeAddIceCandidate.length === 0) {
+  if (!nativeAddIceCandidate) {
     return;
   }
   window.RTCPeerConnection.prototype.addIceCandidate =
